documentation/examples/simpleResourceSample.json

```diff
@@ -0,0 +1,307 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SwaggerSample/sampleResources": {
+      "get": {
+        "tags": [
+          "SwaggerSample"
+        ],
+        "operationId": "SampleResources_ListByResourceGroup",
+        "description": "Delete an instance of your sample resource",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of sample resources in the given resource group",
+            "schema": {
+              "$ref": "#/definitions/SampleResourceList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.SwaggerSample/sampleResources": {
+      "get": {
+        "tags": [
+          "SwaggerSample"
+        ],
+        "operationId": "SampleResources_List",
+        "description": "List all sampleResource instances in the given subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of sample resources in the given subscription",
+            "schema": {
+              "$ref": "#/definitions/SampleResourceList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "description": "Common resource definition. TODO: move to external reference/x-swagger common definition",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "tags": {
```

Comment:
"sku" is missing in this example. As per V2 of ARM sku is a top level property of the Resource.
One more thing to note: I have seen variations of the Sku definition. Most of the Sku objects have properties like 
- name
- tier

However, some RPs have properties like
- family 

in the sku definition.

It would be nice if we can standardize on this aspect as well as it is coming from ARM.